docs/appendix.rst

```diff
@@ -547,6 +547,55 @@ Python versions). If this issue shows up, try installing the
 
 .. _Binary lxml package for Windows: https://www.lfd.uci.edu/~gohlke/pythonlibs/#lxml
 
+ConnectionError raised with [SSL: UNSUPPORTED_PROTOCOL]
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+On newer versions of the operating system running the pywbem client,
+communication with the WBEM server may fail with::
+
+    pywbem.exceptions.ConnectionError: SSL error <class 'ssl.SSLError'>:
+      [SSL: UNSUPPORTED_PROTOCOL] unsupported protocol (_ssl.c:1056)
+
+For example, this happened after an upgrade of the client OS to Debian buster
+using Python 3.7, with OpenSSL 1.1.1d.
+
+This is an error that is created by the OpenSSL library and handed back up to
+the SSL module of Python which hands it up to pywbem. The error indicates that
+OpenSSL and the WBEM server do not agree about which SSL/TLS protocol level to
+use.
+
+Pywbem specifies SSL parameters such that the highest SSL/TLS protocol version
+is used that both the client and server support. Thus, pywbem does not put any
+additional restrictions on top of OpenSSL.
+
+On Debian buster, OpenSSL 1.1.1d increased the security settings to require at
```

Comment:
Remove first word " On "

Reply:
DONE.
Changed to: "Debian buster includes OpenSSL 1.1.1d and increased its security settings to require at least TLS 1.2".